src/state/apis/covalent/validators.ts

```diff
@@ -221,6 +221,7 @@ export const parseToNftItem = (
         medias,
         price: null, // Covalent doesn't provide spot pricing for NFT items
         rarityRank: null, // Covalent doesn't provide rarity rank
+        description: nftData.external_data?.description || '',
```

Comment:
```suggestion
        description: nftData.external_data?.description ?? '',
```

Reply:
ditto empty string needs `||`

Reply:
it would be just to catch the undefined, if description is an empty string, no need to evaluate falsey and return another empty string.

Reply:
Ditto will tackle in the last PR of the stack